py.typed missing in source distribution

**Describe the bug**

The `py.typed` file exists in the `tortoise` package directory in the repository and is supposed to be included in the distribution:

https://github.com/tortoise/tortoise-orm/blob/bda38c6d955bf1499a054aba2f99b972f7268121/setup.py#L38

However in the source distribution found on PyPI, it is missing, causing mypy to ignore the inline type annotations.

https://files.pythonhosted.org/packages/86/8d/a07de37e433fb98accaaaf837f5ee9778366303a36587f7bfb4586a83f2a/tortoise-orm-0.16.13.tar.gz

(link from PyPI's "Download Files" section)

![image](https://user-images.githubusercontent.com/19366641/85151138-55ccd500-b24b-11ea-801a-96a5b79f85a2.png)


**To Reproduce**

```console
$ pip3 install --user tortoise-orm mypy
...
$ python3 -V                                                     
Python 3.8.3
$ mypy -V
mypy 0.780
$ python3 -c "import tortoise; print(tortoise.__version__)"
0.16.13
$ cat tortoise_mypy_test.py       
from tortoise import Tortoise

Tortoise.init()
$ mypy tortoise_mypy_test.py 
tortoise_mypy_test.py:1: error: Skipping analyzing 'tortoise': found module but no type hints or library stubs
tortoise_mypy_test.py:1: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 1 source file)
$ touch ~/.local/lib/python3.8/site-packages/tortoise/py.typed
$ mypy tortoise_mypy_test.py                                  
Success: no issues found in 1 source file
```

**Expected behavior**

`py.typed` should exist in the source distribution by default.

**Additional context**

N/A.

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -2,3 +2,4 @@ include LICENSE.txt
 include README.rst
 include requirements.txt
 include requirements-accel.txt
+include tortoise/py.typed

--- a/setup.py
+++ b/setup.py
@@ -35,7 +35,6 @@ setup(
     zip_safe=True,
     # Include additional files into the package
     include_package_data=True,
-    package_data={"tortoise": ["py.typed"]},
     # Details
     url="https://github.com/tortoise/tortoise-orm",
     description="Easy async ORM for python, built with relations in mind",

